configure.ac

```diff
@@ -64,7 +64,13 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([], [[
 #if !defined(NC_HAS_ZSTD) || NC_HAS_ZSTD == 0
       choke me
 #endif]])], [nc_has_zstd=yes], [nc_has_zstd=no])
+
 if test "x$nc_has_zstd" = xyes; then
+
+   if test "x${HDF5_PLUGIN_PATH}" = "x"; then
+      AC_MSG_ERROR([libnetcdf was built with zstd support, but HDF5_PLUGIN_PATH is not set.  Please set the HDF5_PLUGIN_PATH environmental variable, and re-run configure])
```

Comment:
I don't think you can make this an AC_MSG_ERROR, though it can be a AC_MSG_WARNING.

The reason is, if they did install in the default HDF5 plugin directory (which many will do), then they will not have to set HDF5_PLUGIN_PATH.

Reply:
Errors turned into warnings, in addition to some other stuff.  I'll get this (and the PR nested within) merged soon and hopefully push out the release tomorrow unless there's something unexpected that crops up.